bcncita/cita.py

```diff
@@ -386,7 +386,7 @@ def certificados_ue_step2(driver: webdriver, context: CustomerProfile):
 
     # Select doc type
     if context.doc_type == DocType.PASSPORT:
-        driver.find_element_by_id("rdbTipoDocPasDdi").send_keys(Keys.SPACE)
+        driver.find_element_by_id("rdbTipoDocPasDdiDdi").send_keys(Keys.SPACE)
```

Comment:
Apologies, I only tested it for Certificados UE! The reason I proposed it was because the radio button didn't seem to be selected, however it still works. There is an error in the console which is shown, but it doesn't seem to have any negative repercussions. Please ignore the request!